### PR TITLE
Pull request for review your own project from trials. My own comments added to code

### PR DESCRIPTION
**Pull request by TechnoAndy**
Add inters code

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (TechnoAndy) [inline .github/workflows/linters.yml:56]: All linters have passed

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+A shout out to the most amazing collaboration team I have ever worked with.
+
+📝 License
+This project is MIT licensed.
```
Comment (TechnoAndy) [inline README.md:29]: Good detail in READme file

File: style.css [hunk 3]
```diff
@@ -1,97 +1,113 @@
...
+  margin: auto;
 }
+
+/* .media-icons {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+    padding: 12px 25px 35px 0;
+  } */
```
Comment (TechnoAndy) [inline style.css:113]: Please do not leave unused code as comments on your code. Unused code as comments makes your code look messy.

**Review by frank1738 — APPROVED** [review]
Hi @TechnoAndy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)


## Highlights
✅ Correct use of Github workflow
✅ No linter error
✅ Provided reason for required change 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**